Makefile

```diff
@@ -25,7 +26,9 @@ swagger:
 build-multistage:
 	docker build -t $(tag) -f ./Dockerfile .
 
-build: bin/$(bin)
+build:
+	rm -f ./bin/$(bin)
+	$(MAKE) name=${name} version=${version} bin=goldpinger pkg=${pkg} goos=linux && \
```

Comment:
Hey I'm not sure why change this ? The binary only needs to be rebuilt if any of the `$(files)` is changed, and also this introduces repetition with the `bin/$(bin)` target.

Reply:
Thank you for pointing out, to be honest I was not sure about this place.

Actually, I agree with you – just found an approach that is much better 😄(I'm not a pro in `Makefiles`). I'll force-push (and also rebase) the branch.

About the removal of a binary – I thought as we can't gurantee that binary is compiled for the `linux` OS may be it's better to remove it everytime. Otherwise we'll get a broken Docker image. I researched this topic a little bit – looks like it's ok to let the user decide what to do in this case, so I'll remove this part as well 😄

Reply:
Great 👍